src/data/organizations.js

```diff
@@ -0,0 +1,26 @@
+export default {
+  org1: {
+    ID: 'org1',
+    name: 'Organization 1',
```

Comment:
To make the sample data a little more meaningful, can you give random organization names (whether real or made up)? e.g. Acme, Cisco, Gmail, etc.